config/locales/en.yml

```diff
@@ -2,9 +2,10 @@ en:
   privacy_notice:
     label: Privacy
   check_your_answers:
-    title: Check your answers before sending your application
+    title: Are you ready to send your application?
+    hint: Check your answers first.
```

Comment:
This is intended to appear immediately below the H1. Are any changes required to other files to make this work?